### PR TITLE
Feat: completed Card component

### PR DESCRIPTION
**Pull request by vinc3nati**

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]

**Review by romabulani — APPROVED** [review]

File: components/Button/button.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <button class="btn primary">Primary</button>
     <button class="btn secondary">Secondary</button>
     <button class="btn error">Error</button>
-    <button class="btn success">Success</button>
+    <button class="btn success disabled" disabled>Success</button>
```
Comment (OmKakatkar) [inline components/Button/button.html]: Just a small suggestion, you can use the `:disabled` selector instead of a class here for styling in CSS.

**Review by OmKakatkar — APPROVED** [review]

File: components/Button/button.html (hunk 1 above)
Comment (vinc3nati) [reply components/Button/button.html]: Sure, on it!

**Review by Shah-Arsalan — COMMENTED** [review]
Good to go

**Review by PeeyushPrashant — COMMENTED** [review]
Looks Good

**Pull request merged**